pyoverkiz/client.py

```diff
@@ -823,7 +811,7 @@ async def check_response(response: ClientResponse) -> None:
             if "Not such token with UUID: " in message:
                 raise NotSuchTokenException(message)
 
-        raise Exception(message if message else result)
+        raise UnknownException(message or result)
```

Comment:
Sourcery - Raise a specific error instead of the general Exception.
Sourcery - Simplify if expression by using or